data/bbox2.geojson

```diff
@@ -0,0 +1,5 @@
+{ type: 'Feature',
+  properties: {},
+  geometry: 
+   { type: 'Point',
+     coordinates: [ -90.63720703125, 39.28329180414049 ] } }
```

Comment:
These files ☝️ should have the `.json` extension, be valid JSON, and use the standard JSON formatting for this repo.

If you change the extension and then run `npm run jsonlint`, everything should be 👌 .